README.pt-br.md

```diff
@@ -1434,6 +1551,277 @@ configure :production, :test do
 end
 ```
 
+Você pode acessar essa opções através de `settings`:
+
+```ruby
+configure do
+  set :foo, 'bar'
+end
+
+get '/' do
+  settings.foo? # => true
+  settings.foo  # => 'bar'
+  ...
+end
+```
+
+### Configurando Proteção de Ataques
+
+Sinatra usa
+[Rack::Protection](https://github.com/sinatra/sinatra/tree/master/rack-protection#readme)
+para defender sua aplicação contra ataques comuns e oportunistas. Você pode
+facilmente desativar esse comportamento (o que abrirá sua aplicação para uma
+tonelada de vulnerabilidades):
+
+```ruby
+disable :protection
+```
+
+Para pular uma única camada de defesa, passe `protection` para as opções de um
+hash:
+
+```ruby
+set :protection, :except => :path_traversal
+```
+Você também pode user um array para desativar uma lista de proteções:
+
+```ruby
+set :protection, :except => [:path_traversal, :session_hijacking]
+```
+
+Por padrão, Sinatra só vai setar a sessão baseada em proteção se `:sessions`
+estiver ativada. Veja '[Using Sessions](#using-sessions)'. Algumas vezes você
+pode querer setar a sessão "fora" da aplicação Sinatra, como no config.ru ou uma
+instância separada do `Rack::Builder`. Nesse caso você ainda pode configurar a
+sessão baseada em proteção passando a opção `:session`:
+
+```ruby
+set :protection, :session => true
+```
+
+### Configurações Disponíveis
+
+<dl>
+  <dt>absolute_redirects</dt>
+    <dd>
+      Se desativado, Sinatra permitirá redirecionamentos relativos, no entanto,
+      Sinatra deixará de estar em conformidade com a RFC 2616 (HTTP 1.1), que só
+      permite redirecionamentos absolutos.
+    </dd>
+    <dd>
+      Ative se a sua aplicação estiver rodando em um proxy reverso que não foi
+      configurado corretamente. Observe que o <tt>url</tt> helper ainda vai
+      produzir URLs absolutadas, a menos que voc&e passe <tt>false</tt> como
```

Comment:
We can use "absolutas" because it's a common idiom too

```suggestion
      produzir URLs absolutas, a menos que você passe <tt>false</tt> como
```